editor/src/clj/editor/prefs.clj

```diff
@@ -169,6 +169,8 @@
                :properties
                {:load-external-changes-on-app-focus {:type :boolean
                                                      :default true}
+                :save-on-app-focus-lost {:type :boolean
+                                         :default true}
```

Comment:
Shouldn't the default be false?

Reply:
I don't know, to be honest. 

As a strange side effect, it saves on every popup "open"